articles/quickstart/native/ionic4/01-login.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to add user login to an Ionic 4 application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ionic4
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+<%= include('../../../_includes/_callback_url') %>
+
+The **Callback URL** to be used for your application includes your app's package ID which is found in the `config.xml` file for your app.
+
+Go to the <a href="${manage_url}/#/applications/${account.clientId}/settings">Application Settings</a> section in your Auth0 dashboard and set your **Callback URL** in the **Allowed Callback URLs** box.
+
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Callback URL** to
+
+```bash
+# replace YOUR_PACKAGE_ID with your app package ID
+YOUR_PACKAGE_ID://${account.namespace}/cordova/YOUR_PACKAGE_ID/callback
+```
+
+Replace `YOUR_PACKAGE_ID` with your application's package name, available as the `applicationId` attribute in the `app/build.gradle` file.
```

Comment:
I cannot find a `app/build.gradle` file, however just previously I was instructed to find my app’s package ID from the `config.xml` file. 

This section could probably be summed up in 3 steps
1. How to retrieve `YOUR_PACKAGE_ID`
2. How to construct the CallbackURL using YOUR_PACKAGE_ID.
3. Where to configure your CallbackURL